src/WishlistView/WishlistView.scss

```diff
@@ -0,0 +1,35 @@
+.WishlistView {
+
+  .WishlistViewItem {
+    margin: 20px 0;
+  }
+
+  .ImageBlock {
+    position: relative;
+    width: 200px;
+  }
+  .ImageBlock--Overlay {
+    position: absolute;
+    top: 25%;
+    left: 0;
+    width: 100%;
+  }
+
+  .ImageBlock--OverlayText {
+    color: white;
+    font-size: 20px;
+    background: rgb(0, 0, 0); /* fallback color */
```

Comment:
Mixing and matching color names above "white" with rgb values here, decide on one approach and use it throughout.